operator/src/main/java/oracle/kubernetes/operator/ConfigOverrideDistributionStrategy.java

```diff
@@ -0,0 +1,33 @@
+// Copyright (c) 2020, Oracle Corporation and/or its affiliates.
+// Licensed under the Universal Permissive License v 1.0 as shown at https://oss.oracle.com/licenses/upl.
+
+package oracle.kubernetes.operator;
+
+public enum ConfigOverrideDistributionStrategy {
+  DYNAMIC, ROLLING;
+
+  public static final ConfigOverrideDistributionStrategy DEFAULT = DYNAMIC;
+}
+
+
+/*
```

Comment:
This looks like implementation notes. Does it belong here?

Reply:
Absolutely not. Oops. Deleting.

Reply:
Nope.